PLEP-0008.rst

```diff
@@ -0,0 +1,166 @@
+PLEP-0008 – Calendar Versioning
+===============================
+
++-------------------+---------------------------------------------+
+| PLEP              | number                                      |
++===================+=============================================+
+| title             | PlasmaPy Versioning and Releases            |
++-------------------+---------------------------------------------+
+| author(s)         | Nick Murphy                                 |
++-------------------+---------------------------------------------+
+| contact email     | namurphy@cfa.harvard.edu                    |
++-------------------+---------------------------------------------+
+| date created      | 2022-11-22                                  |
++-------------------+---------------------------------------------+
+| date last revised | 2022-11-29                                  |
++-------------------+---------------------------------------------+
+| type              | process                                     |
++-------------------+---------------------------------------------+
+| status            | discussion                                  |
++-------------------+---------------------------------------------+
+| DOI               |                                             |
+|                   |                                             |
++-------------------+---------------------------------------------+
+
+Brief Description
+-----------------
+
+PlasmaPy will use `calendar versioning`_ starting with the first release
+of 2023.
+
+.. note::
+
+    This PLEP supersedes |PLEP 5|, which is no longer in effect.
+
+Versioning Specification
+------------------------
+
+PlasmaPy final releases will use `calendar versioning`_ of the
+form ``YYYY.0M.MICRO``, where:
+
+* The major release number ``YYYY`` is the year,
+* The minor release number ``0M`` is the zero-padded month, and
+* The bugfix release number ``MICRO`` will start from ``0`` and be
+  incremented by one for each bugfix release.
+
+For example, the first release in April 2023 would be ``2023.04.0``,
+with ``2023.04.1`` as the first bugfix release of the ``2023.04.*``
+series.
+
+Bugfix releases shall contain no changes to the API.
+
+Tradeoff Analysis
+-----------------
+
+Advantages of adopting calendar versioning
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+* The age of a release can be gauged directly from the version number.
```

Comment:
Not sure the best way to make the comment, but also easier to traceback the version when cited in a journal article.